parm/parm_fv3diag/field_table_thompson_noaero_tke_GOCART

```diff
@@ -0,0 +1,192 @@
+# added by FRE: sphum must be present in atmos
```

Comment:
This file is not being used, and probably isn't needed. global-workflow has a separate field table (field_table.aero) that is just appended when DO_AERO is on.

Reply:
I don't disagree, but the UFS right now is designed towards the static RTs and not a more dynamic system where files are determined by settings at runtime. We could do it the UFS way, but it makes for a big messy decision tree to choose the appropriate file (though it looks like maybe the choices have been narrowed in UFS, at least for the field table). diag_table has the same issue (so does nems.configure, but I don't think there is any way around that one since pieces just can't be cat'd together).

For now, let's keep what we have and just make sure the difference between the GOCART one and the regular one matches what is in the field_table.aero. Long-term g-w needs to have a dialog with UFS about how some of these things are set up so we can stop reinventing everything all the time.

Reply:
Confirm that the field table that results from the system matches this file, then delete it. If they don't match, Point me to the field_table that is produced and I'll dig in.

Reply:
@WalterKolczynski-NOAA Good thing we checked... we do have differences in the "surface_value"s:To see for example: 
diff -w -y /scratch2/NCEPDEV/climate/Jessica.Meixner/p8b/prtests/coupled/aero2d/RUNDIRS/aero2d/2013040100/gfs/fcst.64261/field_table /scratch2/NCEPDEV/climate/Jessica.Meixner/p8b/prtests/coupled/global-workflow/parm/parm_fv3diag/field_table_thompson_noaero_tke_GOCART

I can make a new run and make sure I didn't grab an old or outdated directory, but pretty sure we might have found something that needs to be fixed